samples/Java/azure-servicebus/ManagingTopicRules/src/main/java/com/microsoft/azure/servicebus/samples/managingtopicrules/ManagingTopicRules.java

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.servicebus.samples.managingtopicrules;
+
+import com.microsoft.azure.servicebus.*;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import com.microsoft.azure.servicebus.rules.CorrelationFilter;
+import com.microsoft.azure.servicebus.rules.RuleDescription;
+import com.microsoft.azure.servicebus.rules.SqlFilter;
+import com.microsoft.azure.servicebus.rules.SqlRuleAction;
+import org.apache.commons.cli.*;
+import org.apache.log4j.*;
+
+import java.time.Duration;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class ManagingTopicRules {
+    // Connection String for the namespace can be obtained from the Azure portal under the
```

Comment:
java files are present in both the PRs.
#90 and #86. Only one of them can be checked in.
So could you please close #86 and address all its comments in this PR?

Reply:
Did that